server/Db/projectDb.js

```diff
@@ -23,7 +23,6 @@ function deleteEvent (id, db = connection) {
 }
 
 function addUserToEvent (id, eventId, db = connection) {
-  console.log(id, eventId)
```

Comment:
👏🏻 Good catch; removing obsolete debug statements makes it easier to troubleshoot issues later on.